client/src/Views/Home.jsx

```diff
@@ -1,23 +1,47 @@
 import React, { Component } from 'react';
 import SearchForm from '../Components/SearchForm';
 import AddBook from '../Components/AddBook';
-import { Card } from 'semantic-ui-react';
+import BookElement from '../Components/BookElement';
+import { Card  } from 'semantic-ui-react';
 
 class Home extends Component {
-state = { data: [] };
 
-changeData = data => {
-this.setState({ data });
+state = { findBooks: [],
+          allBooks: []
+        };
+    
+findData = findBooks => {
+  this.setState({ findBooks });
+}
+
+showAllData = allBooks => {
+  this.setState({ allBooks });
 }
 
 render(){
   return (
     <>
-      <SearchForm changeData={this.changeData} />
+      <SearchForm findData={this.findData} />
+      {this.state.findBooks.map(book => {
+        return (
+        <Card color="blue" key={book._id}>
+        <Card.Content>
+          {"Title: " + book.title.toUpperCase() + "  *  Author: " + book.author.toUpperCase() + "  *  Year: " + book.year + "  *  Genre: " + book.genre.toUpperCase()}
+        </Card.Content>       
+        </Card>
```

Comment:
Przeniósłbym to do innego komponentu, i troszkę upiększył od strony wizualnej. Card może przyjąć nie tylko content ale też header, description meta. Dojdzie później jeszcze obrazek, także będzie co robić w tym komponencie.

https://react.semantic-ui.com/views/card/